app/helpers/alchemy/admin/base_helper.rb

```diff
@@ -376,7 +376,7 @@ def render_hint_for(element)
 
         content_tag :span, class: "hint-with-icon" do
           render_icon("question-circle") +
-            content_tag(:span, element.hint.html_safe, class: "hint-bubble")
+            content_tag(:span, element.hint.try(:html_safe), class: "hint-bubble")
```

Comment:
We should add the support for `false` into [`has_hint?` method](https://github.com/AlchemyCMS/alchemy_cms/blob/master/lib/alchemy/hints.rb#L47-L49) instead, so that we do not even render the hint bubble.

Reply:
Actually, it does that already. `has_hint?` checks for presence, which `false` isn't. Reverted.